dashboard/app/controllers/registrations_controller.rb

```diff
@@ -445,6 +445,7 @@ def existing_account
       :provider,
       :us_state,
       :country_code,
+      :ai_rubrics_disabled,
```

Comment:
Should AI features be opt in or opt out? Full disclosure, I think they should be opt-in but I'd want @tess323 to make the final decision there

Reply:
I would like it to be opt in, but it felt like it was opt out from the descriptions. Will the AI popup opt people in?

Reply:
The rubric in the level could be used for both opt-in and opt-out I think. By "descriptions", do you mean a product spec?

Reply:
Acknowleding that we got an answer in Slack to be opt-out here